pynest/nest/lib/hl_api_connections.py

```diff
@@ -453,15 +452,15 @@ def _is_subnet_instance(gids):
     """
 
     try:
-        # Turn off _deprecation_warning as users shouldn't change
-        # implementation of this function, it is done by the developers
-        deprecation_bool = hlh._deprecation_warning['GetChildren']
-        hlh._deprecation_warning['GetChildren'] = False
+        # Turn off deprecation warning to avoid confusing users with
+        # internals.
+        # deprecation_status = suppress_deprecation_warning('GetChildren')
 
-        GetChildren(gids)
+        with SuppressedDeprecationWarning(['GetChildren']):
```

Comment:
Where is this defined? Couldn't find it in ``hl_api_helper``
**EDIT:** ok, just below ^^